uitools/import/Esri/ArcGISRuntime/Toolkit/Controller/FloorFilterController.qml

```diff
@@ -23,6 +23,90 @@ QtObject {
 
     property GeoView geoView
 
+    property Connections geoViewConnections: Connections {
+        target: geoView
+        function onViewpointChanged() {
+            tryUpdateSelection()
+        }
+        // manually set true when floorManager is loaded
+        enabled: false
+    }
```

Comment:
🔧 Move `geoViewConnections` into the `internal` object, otherwise it will be public in the controller interface..